problems/removeNumbersAtOddIndices.js

```diff
@@ -2,8 +2,11 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+function removeNumbersAtOddIndices(arr) {
+    let newArr = arr.filter(i => {
```

Comment:
Close! You array method callbacks will always take the element as it's first argument. If you want the index, you need to add a second argument. 
filter((el, i) => { etc...